### ISSUE TITLE
Crash when doing a DHT lookup

### ISSUE THREAD

**Issue by devos50** [Tribler/tribler#3952]
Available in the reporter (number 16872) There's a crash when doing a DHT lookup:

```
Traceback (most recent call last):
File "TriblerGUI\event_request_manager.py", line 130, in on_read_data
RuntimeError: [Failure instance: Traceback: <type 'exceptions.KeyError'>: 2337470787L
lib\threading.py:801:__bootstrap_inner
lib\threading.py:754:run
lib\site-packages\twisted\_threads\_threadworker.py:46:work
lib\site-packages\twisted\_threads\_team.py:190:doWork
--- <exception caught here> ---
lib\site-packages\twisted\python\threadpool.py:250:inContext
lib\site-packages\twisted\python\threadpool.py:266:<lambda>
lib\site-packages\twisted\python\context.py:122:callWithContext
lib\site-packages\twisted\python\context.py:85:callWithContext
Tribler\Core\APIImplementation\LaunchManyCore.py:752:sesscb_states_callback
Tribler\community\triblertunnel\community.py:491:monitor_downloads
Tribler\pyipv8\ipv8\messaging\anonymization\hidden_services.py:128:do_raw_dht_lookup
Tribler\pyipv8\ipv8\messaging\anonymization\community.py:138:select
]
```

**Comment by qstokkink** [comment]
That line should become:

```python
return self.community.active_data_circuits().get(circuit_id, None)
```

**Comment by xoriole** [comment]
Apparently, this issue is still prevalent in Tribler 7.1.0. See error report [#17133](http://reporter.tribler.org/stacktrace/17133) of [2018-10-17](http://reporter.tribler.org/crashreport_daily/2018-10-17)
Re-opening again.

**Comment by egbertbouman** [comment]
@xoriole It looks like the fix hasn't made it into the 7.1.0 release.

**Comment by xoriole** [comment]
@egbertbouman yes, you're right
